### PR TITLE
More test cases have been added in `test_clebsch_gordan.py`

### PR DESCRIPTION
**Pull request by mamakancha**
As mentioned earlier in the file, more tests cases were required to be added using the source provided in the file.
Test cases have been added in the functions `test_clebsch_gordan1()` and `test_clebsch_gordan2()`

### CODE REVIEW

**Comment by mamakancha** [comment]
@jksuom Please review.

**Comment by jksuom** [comment]
It seems that the tests were organized according to a plan:
test 1: j1 = 1/2, test 2: j1 = 1

So maybe we should have:
test 3 : j1= 3/2, test 4: j1 = 2 and possibly test 5: j1 = 5/2.

File: sympy/physics/tests/test_clebsch_gordan.py [hunk 1]
```diff
@@ -3,9 +3,7 @@
         racah, dot_rot_grad_Ynm, Wigner3j, wigner_3j)
 from sympy.core.numbers import Rational
 
-# Todo: more tests should be added from:
-# http://en.wikipedia.org/wiki/Table_of_Clebsch-Gordan_coefficients
-
+#for test cases, refer : https://en.wikipedia.org/wiki/Table_of_Clebsch%E2%80%93Gordan_coefficients
```
Comment (jksuom) [inline sympy/physics/tests/test_clebsch_gordan.py]: A space after #

**Comment by mamakancha** [comment]
@jksuom Please review :)

**Comment by mamakancha** [comment]
@jksuom Can you let me know what is leading to the failure?

**Comment by jksuom** [comment]
Some time yesterday theano 0.8.2 was updated to 0.9.0. All theano jobs have been failing thereafter. It seems that .travis.yml has to be modified to load the old version.

**Comment by mamakancha** [comment]
Could I contribute to modifying `.travis.yml` ? If yes then if you could guide me through the same that would be great!

**Comment by jksuom** [comment]
I believe that one should first investigate if the test itself should be modified because of some intentional change in theano. If there is clearly a bug in theano, then .travis.yml should be modified.

**Comment by mamakancha** [comment]
@jksuom I have submitted a new PR fixing the issue of `theano` as also mentioned in **gitter**. Please see #12403

**Comment by jksuom** [comment]
Thanks! This is ready.

**Pull request merged**